File: core/app/datasource/src/main/java/io/openk9/datasource/web/dto/TabResponseDTO.java
package io.openk9.datasource.web.dto;

import java.util.List;
import java.util.Map;

public record TabResponseDTO(
	String label, List<TokenTabResponseDTO> tokens,
	List<SortingResponseDTO> sortings,
	Map<String, String> translationMap) {
}

File: js-packages/admin-ui/src/openapi-generated/models/TabResponseDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { SortingResponseDTO } from './SortingResponseDTO';
import type { TokenTabResponseDTO } from './TokenTabResponseDTO';

export type TabResponseDTO = {
    label?: string;
    tokens?: Array<TokenTabResponseDTO>;
    sortings?: Array<SortingResponseDTO>;
    translationMap?: Record<string, string>;
};
